app/build.gradle

```diff
@@ -353,7 +353,7 @@ distributions {
                 exclude 'es.jar'
                 exclude 'es-*.jar'
             }
-            into("lib") {
+            into("lib/main") {
```

Comment:
Would it be possible to change how we define the classpath to make this structure change unnecessary?